JPF/bad_buildings.sql

```diff
@@ -117,20 +151,22 @@ select
 	hb.lot as lot,
 	hb.bin as bin,
 	hb.zip as zip,
```

Comment:
Just aliasing columns to avoid typing out the table name feels like overkill, especially since you've already aliased the table name to shorthand.